core/app/models/workarea/data_file/csv.rb

```diff
@@ -5,6 +5,7 @@ def import!
         index = 2 # start at 1 and skip headers
         failed_new_record_ids = []
         options = Workarea.config.csv_import_options.merge(headers: true)
+        options[:encoding] ||= 'bom|utf-8'
```

Comment:
Why wouldn't this be in the default value of `Workarea.config.csv_import_options` instead of setting here?